api-gateway/src/main/java/io/eventuate/examples/tram/sagas/ordersandcustomers/apiagateway/common/Money.java

```diff
@@ -0,0 +1,61 @@
+package io.eventuate.examples.tram.sagas.ordersandcustomers.apiagateway.common;
+
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.apache.commons.lang.builder.ToStringBuilder;
+
+import java.math.BigDecimal;
+
+public class Money {
```

Comment:
Please don't copy/paste classes. Move into a shared module, e.g. `common-domain`

Reply:
There is already common module with that single class, but also with jpa deps.
I've copied classes because ftgo-gateway does it. I will remove copy/paste.